swarm/pss/pss_test.go

```diff
@@ -951,7 +953,7 @@ func worker(id int, jobs <-chan Job, rpcs map[discover.NodeID]*rpc.Client, pubke
 // nodes/msgs/addrbytes/adaptertype
 // if adaptertype is exec uses execadapter, simadapter otherwise
 func TestNetwork2000(t *testing.T) {
-	//testutil.EnableMetrics()
+	//enableMetrics()
```

Comment:
remove comment?

Reply:
Wouldn't that be better with a flag then

Reply:
IDEs are hard to configure to use custom flags for every test, I personally find it easier to uncomment the line.

Reply:
I am fine with this, but I have had comments on my PRs to have such comments removed. Maybe we should discuss it in a roundtable?

Reply:
@holisticode sure, let's discuss this. I agree in general that we should not keep dead code, especially in the production code, however I already removed this once from tests, and then we had to introduce it yet again with @nolash . I also think this specific line will be helpful for other tests in the future.

I have definitely been against dead code in the production code in the past, and if you feel strongly that this is the same, I will be happy to remove it! :)

Reply:
No strong feelings, just aiming at consistent handling.